app/controllers/api/v1/item_merchant_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::ItemMerchantController < ActionController::API
+  def index
+    item = Item.find(params[:item_id])
+    render json: MerchantSerializer.new(item.merchant)
+  end
+end
```

Comment:
May want to reconsider how I setup item merchant, but for now using this controller and index action.